lib/rucio/daemons/reaper/reaper.py

```diff
@@ -141,8 +141,13 @@ def delete_from_storage(heartbeat_handler, hb_payload, replicas, prot, rse_info,
     rse_id = rse_info['id']
     noaccess_attempts = 0
     pfns_to_bulk_delete = []
+
+    refresh_start_time = time.time()
```

Comment:
By having this before `prot.connect()`, the `elapsed_time` for the first replica will always be longer than all other replicas in the `for replica in replicas` loop - it should be moved under `for replica in replicas:`